pkg/server/plugin/nodeattestor/aws/iid.go

```diff
@@ -183,6 +198,10 @@ func (p *IIDAttestorPlugin) Configure(ctx context.Context, req *spi.ConfigureReq
 		return resp, err
 	}
 
+	if config.SkipEC2AttestCalling {
+		p.log.Warn("skip_ec2_attest_calling is a deprecated flag and will be ignored")
```

Comment:
can you add a unit test that sets this deprecated flag just to make sure we don't blow up (e.g. the logger is somehow not set so logging causes a panic).